### DISCUSSION TITLE
PTZOptics help

### DISCUSSION THREAD

**Discussion by bardnaut** [bitfocus/companion#2904] [Q&A]
I want to add a button to Companion that enables camera auto tracking. I placed a call to PTZOptics and they said it can be done with an http script. I generally know what he was talking about but I don't know how to write it or program it. Can anyone help with this please?

**Comment by ploveman** [comment]
@bardnaut It looks like the PTZOptics module supports enabling/disabling auto-tracking, have you tried using that?

![image](https://github.com/bitfocus/companion/assets/46356110/05eafba5-25c1-4706-b0e8-549fcabe12b9)
